README.md

```diff
@@ -84,22 +84,32 @@ this to your [coc.nvim configuration file][vim_coc_nvim_config_file]:
 ```
 #### Using NeoVim's native language server
 
-```vim
+This requires [`nvim-lspconfig`](https://github.com/neovim/nvim-lspconfig).
+```lua
```

Comment:
Figured since the majority of the snippet is `lua`, makes more sense to highlight that